lib/msal-common/src/client/AuthorizationCodeClient.ts

```diff
@@ -173,6 +174,11 @@ export class AuthorizationCodeClient extends BaseClient {
         parameterBuilder.addGrantType(GrantType.AUTHORIZATION_CODE_GRANT);
         parameterBuilder.addClientInfo();
 
+        if (request.authenticationScheme === AuthenticationType.POP) {
+            const popTokenGenerator = new PopTokenGenerator(this.cryptoUtils);
+            parameterBuilder.addPopToken(await popTokenGenerator.generateCnf());
```

Comment:
Can you factor the `generateCnf` call out to a variable?

Reply:
done.